.buildkite/pipeline.yml

```diff
@@ -168,5 +168,25 @@ steps:
         - exit_status: -1  # Agent was lost
           limit: 2
 
+  - label: ':apple: macOS 10.15 barebones end-to-end tests'
+    depends_on:
+      - cocoa_fixture
+    timeout_in_minutes: 60
+    agents:
+      queue: opensource-mac-cocoa
+    plugins:
+      artifacts#v1.3.0:
+        download: ["features/fixtures/macos/output/macOSTestApp.zip"]
+    commands:
+      - bundle install
+      - bundle exec maze-runner
+        features/barebone_tests.feature
+        --farm=local
+        --os=macos
+        --os-version=10.15
+        --app=macOSTestApp
+        --tags='not @skip_macos'
+        --fail-fast
+
   - label: 'Conditionally trigger full set of tests'
-    command: sh -c .buildkite/pipeline_trigger.sh
+    command: sh -c .buildkite/pipeline_trigger.sh
```

Comment:
Quite a few tools (linters etc.) complain about files that don't end with a newline.. probably better to leave them in 🙃